keymanager/src/crypto/kdf.rs

```diff
@@ -780,7 +780,8 @@ impl Kdf {
         let d2 = new_deoxysii(Keypolicy::MRENCLAVE, MASTER_SECRET_SEAL_CONTEXT);
         let plaintext = d2
             .open(&nonce, ciphertext.to_vec(), additional_data)
-            .expect("persisted state is corrupted");
+            .expect("persisted state is corrupted (if running in SGX mode, check if the CPU had changed; if yes, wipe 'worker-local-storage.badger.db' and resync the node)");
+        // TODO: When logging is added to the keymanager, log the above error.
```

Comment:
```suggestion
```
This should probably be removed, as we are trying to avoid adding logging to the key manager, so that we don't mistakenly log any sensitive information.

Reply:
Done :)  Could you also please check if the other error gets printed now on an SGX machine (I don't have any to test this on)?